config/initializers/0_clowder-config.rb

```diff
@@ -1,19 +1,29 @@
+# frozen_string_literal: true
+
+def build_endpoint_url(endpoint, config)
+  # There is currently no tlsPort or anything similar in clowder-common-ruby
+  return URI::HTTPS.build(host: endpoint&.hostname, port: endpoint&.tlsPort).to_s if config.tls
+
+  URI::HTTP.build(host: endpoint&.hostname, port: endpoint&.port).to_s
+end
 
 if ClowderCommonRuby::Config.clowder_enabled?
 
   config = ClowderCommonRuby::Config.load
 
+  ENV['SSL_CERT_FILE'] = '/cdapp/certs/service-ca.crt' if config.tls
```

Comment:
This is also something that could be provided by the clowder ruby bindings.

Reply:
Do you mean the `'/cdapp/certs/service-ca.crt'` part? We could add a field called `tlsPath` or something to clowder-common-ruby so we can call `config.tlsPath`. That's how app-common-python works.